NewForce-at-Mountwest/bangazon-api-bucatini
# Bf product type controller

# Description

I finished up giving ProductType a class, a controller, and the test to make sure everything is running correctly.


## Related Ticket(s)
5

## Steps to Test
-Check ProductType class to make sure it has the appropriate keys for what is needed from ERD.
-To check the ProductType controller run all test created under ProductTypeTest. To make sure the test are running and checking controller appropriately you can use the assistance of PostMan to see if all GET'S, POST and PUTS are working.


# Checklist:
